budget/budgetRouter.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express')
+const Budget = require('./Budget')
+
+const router = express.Router()
+
+// add endpoints here
+/* EXTRAS
+router
+.route('/')
+.get(get)
+.post(post)
+
+router
+.route('/:id')
+.get(getById)
+.delete(deleteBudget)
+.put(editBudget)
+END EXTRAS */
+
+// const {budget} = req.body
+// const {budgetData} = req.body
+
+// create budget
+router
+  // .route('/')
+  // .get((req, res) => {
+  //   Budget.find()
+  //     .then(budgets => {
+  //       res.status(200).json(budgets)
+  //     })
+  //     .catch(err => {
+  //       res.status(500).json({ errorMessage: 'Error geting budgets', err })
+  //     })
+  // })
+  .post('/budget', (req, res) => {
```

Comment:
The endpoint path should just be `/`, unless it was intentional to have the complete path be `/api/budget/budget`.